docs/content/setup/windows.md

```diff
@@ -18,7 +18,8 @@ After running the installer, the following prompt should appear on the screen:
 ![Windows installer screenshot](../../images/island/others/windows_installer.png "Windows installer screenshot")
 
 1. Follow the steps to complete the installation.
-1. Run the Monkey Island by clicking on the desktop shortcut.
+1. Run the Infection Monkey by clicking on the desktop shortcut. **Note**: If you want Monkey Agents to
+collect more data on the current machine, consider running as Administrator.
```

Comment:
```suggestion
collect more data on the Island machine, consider running as Administrator.
```

Reply:
"current" is better, because the user doesn't need to know what is "the Island" machine.